pkg/types/installconfig.go

```diff
@@ -178,6 +178,15 @@ func (c *InstallConfig) IsOKD() bool {
 	return OKD
 }
 
+// IsBootstrapInPlace returns true if the install-config has been configured for
+// Single Node OpenShift
+func (c *InstallConfig) IsBootstrapInPlace() bool {
+	if c.Platform.Name() == "none" && c.BootstrapInPlace != nil {
+		return true
+	}
+	return false
```

Comment:
nit: This could be simplified.
```suggestion
	return c.Platform.Name() == "none" && c.BootstrapInPlace != nil
```

Reply:
SNO deployments can happen on multiple platform types (None, vsphere and even Baremetal). So, the only valid check would be to check for the presence of BootstrapInPlace config.